batch_import/EXAMPLE.md

```diff
@@ -157,17 +149,23 @@ of the network, displaying all of the devices that it has located. Look through
 Camio supplies a [small directory of video files](https://storage.googleapis.com/camio_firmware_images/batch_import_video_files.zip) 
 that you can use with the `import_video.py` program for testing purposes.  
 
+You can either download them through your browser by click [this link](https://storage.googleapis.com/camio_firmware_images/batch_import_video_files.zip),
```

Comment:
You can either [download the batch_import_video_files](https://storage.googleapis.com/camio_firmware_images/batch_import_video_files.zip) in your browser or run this command: